### PR TITLE
add spitz on chihuahua chain

### PR DESCRIPTION
**Pull request by Swax-from-Stakelab**

### CODE REVIEW

File: osmosis/assetlist.json [hunk 1]
```diff
@@ -1694,6 +1694,49 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/RAPTR.png"
         }
       ] 
+    },
+    {
+      "description": "Spitz on Osmosis",
+      "denom_units": [
+        {
+          "denom": "ibc/1570B676D4F35BACD92F470C01E4FD7B19CA7B8D62DA974A76A59079CEE338E3",
```
Comment (JeremyParish69) [inline osmosis/assetlist.json:1702]: How did you get this ibc denom? It doesn't seem right

File: osmosis/assetlist.json [hunk 2]
```diff
@@ -1694,6 +1694,49 @@
...
+            "path": "transfer/channel-113/factory/chihuahua17dgp6rf7wlhyaxth0j20h33zdv6nqat8xfezvy/spitz"
+          }
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/spitz.png"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/spitz.png"
```
Comment (JeremyParish69) [inline osmosis/assetlist.json:1737]: Please do not upload 2 copies of the image. Please add image sync and point back to the Chihuahua image

File: osmosis/assetlist.json (hunk 1 above)
Comment (Swax-from-Stakelab) [reply osmosis/assetlist.json:1702]: Hi Jeremy; it's my IBC from my wallet, so i don't understand which IBC you need ?

File: osmosis/assetlist.json (hunk 2 above)
Comment (Swax-from-Stakelab) [reply osmosis/assetlist.json]: i modify the path by https://raw.githubusercontent.com/cosmos/chain-registry/master/chihuahua/images/spitz.png

File: osmosis/assetlist.json [hunk 3]
```diff
@@ -1694,6 +1694,49 @@
...
+      "denom_units": [
+        {
+          "denom": "ibc/1570B676D4F35BACD92F470C01E4FD7B19CA7B8D62DA974A76A59079CEE338E3",
+          "exponent": 0,
+          "aliases": [
+            "spitz"
+          ]
+        },
+        {
+          "denom": "Spitz",
```
Comment (JeremyParish69) [inline osmosis/assetlist.json:1709]: the denom_unit defined here is different than the unit from Chihuahua
'SPITZ' != 'Spitz'.
Please try to use the same for both, but even if you don't, the 'display' value must be a defined unit

File: osmosis/assetlist.json [hunk 4]
```diff
@@ -1694,6 +1694,49 @@
...
+            "path": "transfer/channel-113/factory/chihuahua17dgp6rf7wlhyaxth0j20h33zdv6nqat8xfezvy/spitz"
+          }
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/chihuahua/images/spitz.png"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/chihuahua/images/spitz.png"
```
Comment (JeremyParish69) [inline osmosis/assetlist.json:1737]: Please use image sync

```
"image_sync": {
  "chain_name": "chihuahua",
  "base_denom": "factory/chihuahua17dgp6rf7wlhyaxth0j20h33zdv6nqat8xfezvy/spitz"
```
}

File: osmosis/assetlist.json [hunk 5]
```diff
@@ -1694,6 +1694,49 @@
...
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "chihuahua",
+            "base_denom": "factory/chihuahua17dgp6rf7wlhyaxth0j20h33zdv6nqat8xfezvy/spitz",
+            "channel_id": "channel-7"
+          },
+          "chain": {
+            "channel_id": "channel-113",
+            "path": "transfer/channel-113/factory/chihuahua17dgp6rf7wlhyaxth0j20h33zdv6nqat8xfezvy/spitz"
```
Comment (JeremyParish69) [inline osmosis/assetlist.json:1728]: The SHA256 of this path does not yield '157...8E3', but instead 'BA9...3CF'.
Something must not be right, either the channel it's using is different, or the base denom is different or it's not using the slash ('/') characters, or soemthing....

**Comment by JeremyParish69** [comment]
Most of the issues seem to come from trying to add it to Osmosis' assetlist. I recommend not trying to do that here. Soon, Osmosis will automatically update their assetlist here to include all traded tokens, and would be adding this as well if listed on Osmosis Zone.

**Comment by Swax-from-Stakelab** [comment]
I have Update denom spitz, can you confirm please ?

File: chihuahua/assetlist.json [hunk 6]
```diff
@@ -206,6 +206,31 @@
...
+        },
+        {
+          "denom": "spitz",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/chihuahua17dgp6rf7wlhyaxth0j20h33zdv6nqat8xfezvy/spitz",
+      "name": "spitz",
+      "display": "spitz",
+      "symbol": "spitz",
```
Comment (JeremyParish69) [inline chihuahua/assetlist.json:225]: Please make symbol all CAPS

File: osmosis/assetlist.json [hunk 7]
```diff
@@ -1694,6 +1694,49 @@
...
       ] 
+    },
+    {
+      "description": "Spitz on Osmosis",
+      "denom_units": [
+        {
+          "denom": "ibc/1570B676D4F35BACD92F470C01E4FD7B19CA7B8D62DA974A76A59079CEE338E3",
+          "exponent": 0,
+          "aliases": [
+            "spitz"
```
Comment (JeremyParish69) [inline osmosis/assetlist.json:1705]: please remove alias 'spitz' from denom for exponent 0, since it was used as the denom with exponent 6 above.

**Comment by JeremyParish69** [comment]
> I have Update denom spitz, can you confirm please ?

Your update still doesn't address many of the concerns brought up. I highly recommend you remove it from the Osmosis assetlist for now, since that'll only complicate things

**Comment by JeremyParish69** [comment]
PR gone stale. Happy to assist (after addressing my comments ) if there is interest in reopening

**Comment by Swax-from-Stakelab** [comment]
What do you need ? Can you explain me please ? I would like create a token
on Kami’s ! That’s it

Le lun. 18 mars 2024 à 23:05, JeremyParish69 ***@***.***> a
écrit :

**Comment by JeremyParish69** [comment]
I added several comments on how to fix the registration, but all were ignored. The same submission with the comments addressed should be sufficient to register the token.

**Pull request closed without merging**